designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/StatisticError.scala

```diff
@@ -0,0 +1,5 @@
+package pl.touk.nussknacker.ui.statistics
+
+sealed trait StatisticError
+case object DbError  extends StatisticError
```

Comment:
IMO we have two cases here:
1. we cannot provide statistics because we don't have persistent fingerprint 
2. we cannot provide statistics because there was a problem with URL creation

One is related to some unexpected DB errors, and the latter is related to some development bug (is there an expected case where the URL cannot be created?). So, maybe have only one StatisticError implementation, like "CannotGenerateStatistics" (why? it's not sth that the layer above is interested in, but in can be logged, obviously)

Reply:
Done

Reply:
ok, great, but WDYT about sth like that?
```scala
sealed trait StatisticError
object CannotGenerateStatisticsError extends StatisticError
```

Reply:
Done